components/shared/atoms/FormField/FormField.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Field, ErrorMessage } from 'formik';
+import styled from 'styled-components';
+
+import { Input } from 'components/shared/atoms/Input/Input';
+
+const FormInput = styled(Input)`
+  margin-top: 0.5rem;
+`;
+
+const ErrorWrapper = styled.div`
+  font-size: 0.9rem;
+  color: red;
+  padding: 0.5rem 0;
+`;
+
+type FormFiledProps = {
+  name: string;
+  type: string;
+  label: string;
+};
+
+const FormField = ({ name, type, label }: FormFiledProps) => {
+  return (
+    <label htmlFor={name}>
+      {label}
+      <Field type={type} name={name} id={name} data-testid={`input-${name}`} as={FormInput} autoComplete="on" />
```

Comment:
Will approach work with selects, textarea?

Reply:
I'll find out it

Reply:
It requires this approach:
<Field as="select" name="color">
    <option value="red">Red</option>
    <option value="green">Green</option>
    <option value="blue">Blue</option>
</Field>

We can create three different components for select, textarea, input then